coreclient/migrations/20250409073637_add_indexed_keys.sql

```diff
@@ -11,10 +11,9 @@ CREATE TABLE IF NOT EXISTS indexed_keys (
 
 -- Create table to store indices of own keys
 CREATE TABLE IF NOT EXISTS own_key_indices (
-    id INTEGER PRIMARY KEY AUTOINCREMENT,
     key_index BLOB NOT NULL,
-    key_type TEXT CHECK(key_type IN ('user_profile_key')) NOT NULL,
-    FOREIGN KEY (key_index) REFERENCES indexed_keys(key_index)
+    key_type TEXT CHECK(key_type IN ('user_profile_key')) PRIMARY KEY,
```

Comment:
Minor: Maybe primary key should be the first field?

I am not sure actually how useful it is to have this check. Currently, it means this table can only have a single row. In the future, if we add more key index types, we will have to write a migration.

Reply:
Feel free to leave it as is.

Reply:
If it turns out in the future to be too much work, we can still remove the constraint with a migration.